_drafts/practical_zephyr_00_basics.md

```diff
@@ -0,0 +1,695 @@
+)
+
+for entry in ${paths[@]}; do
+    export PATH=$PATH:$entry
+done
+```
+
+> **Note:** Depending on whether or not you want to replace _all_ binaries - including `git` - with Nordic's installation, _append_ or _prepend_ to `PATH`. In the above example, we _append_ and thus use local installations over the installation provided by the toolchain installer, if available.
+
+After sourcing this script, you should now have, e.g., [Zephyr's meta-tool _West_](https://docs.zephyrproject.org/latest/develop/west/index.html) and [CMake](https://cmake.org/) in your path:
+
+```bash
+$ source setup.sh
+$ which cmake
+/opt/nordic/ncs/toolchains/4ef6631da0/bin/cmake
+$ which west
+/opt/nordic/ncs/toolchains/4ef6631da0/bin/west
+```
+
+Only [two environment variables are significant](https://docs.zephyrproject.org/latest/develop/env_vars.html#important-environment-variables) when configuring the toolchain:
+- [`ZEPHYR_SDK_INSTALL_DIR`](https://docs.zephyrproject.org/latest/develop/env_vars.html#envvar-ZEPHYR_SDK_INSTALL_DIR) must be set to the installation directory of the SDK,
+- [`ZEPHYR_TOOLCHAIN_VARIANT`](https://docs.zephyrproject.org/latest/develop/beyond-GSG.html#install-a-toolchain) selects the toolchain to use, e.g., `llvm` or in our case `zephyr`.
+
+> **Note:** Yes, the inconsistent use of the term "SDK" is quite annoying. Nordic uses the term "SDK" to refer to its sources, whereas Zephyr uses _SDK_ to refer to the toolchain. I'm used to Nordic's terminology where an SDK also includes source code.
+
+Once those environment variables are set, we can source the `zephyr-env.sh` that is provided by Zephyr itself, as shown below. This [Zephyr environment script](https://docs.zephyrproject.org/latest/develop/env_vars.html#zephyr-environment-scripts) is the recommended way to load an environment in case Zephyr is not installed globally.
+
+```sh
+# continuing setup.sh from the previous snippet
+export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+export ZEPHYR_SDK_INSTALL_DIR=$ncs_install_dir/toolchains/$ncs_bin_version/opt/zephyr-sdk
+source $ncs_install_dir/$ncs_sdk_version/zephyr/zephyr-env.sh
+west zephyr-export
+```
+
+Instead of setting `ZEPHYR_BASE` manually as, e.g., done by the `env.sh` script generated via the Nordic toolchain manager, `ZEPHYR_BASE` is now set by the `zephyr-env.sh` script (we'll see how `ZEPHYR_BASE` is used when building our application). In addition, this also allows [using `.zephyrrc` files](https://docs.zephyrproject.org/latest/develop/env_vars.html#option-3-using-zephyrrc-files) in your application root directory - a feature we won't cover in this series.
+
+The final command, `west zephyr-export`, is a ["Zephyr extension command"](https://docs.zephyrproject.org/latest/develop/west/zephyr-cmds.html#installing-cmake-packages-west-zephyr-export) that installs the required _CMake_ packages that we'll need in the next steps when using `find_package` to locate Zephyr's sources in a `CMakeLists.txt` file.
```

Comment:
It might be good to note that this does not need to be run every time we source `setup.sh`

Reply:
`west zephyr-export` is actually part of this `setup.sh`, so it is run by that shell script each time we source it. Technically, it might not be necessary.

Reply:
Yep, understood. I'm fine leaving as is. Thanks!